platform/keyword-registry/src/main/java/org/codice/ddf/keyword/KeywordRegistryImpl.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.keyword;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.ResourceBundle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KeywordRegistryImpl implements KeywordRegistry {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(KeywordRegistryImpl.class);
+
+  private Map<String, String> keywords = new HashMap<>();
+
+  private static final String DEFAULT_LOCALE = "KeywordRegistryBundle";
```

Comment:
A better name might be KEYWORD_BASE or something as this is really the base name for a file and not the LOCALE